### ISSUE TITLE
Taskrun is created twice if using build for service

### ISSUE THREAD

**Issue by StrongMonkey** [rancher/rio#475]
Version: v0.5.0-alpha.2

Steps to reproduce:
`rio run https://github.com/rancher/rio-demo`
`rio build`

Output:
```
NAME                                                               SERVICE/STACK                                  REVISION                                   CREATED          SUCCEED   REASON
default/default-distracted-heisenberg9-ee709b9-4e40c-ee709-4e40c   default/distracted-heisenberg9-ee709b9-4e40c   f8fab97fddc8ed5e98e45cd6373ad6feff3197f9   5 minutes ago    True      
default/default-distracted-heisenberg9-ee709-4e40c                 default/distracted-heisenberg9                 f8fab97fddc8ed5e98e45cd6373ad6feff3197f9   5 minutes ago    True    
```

Expected:
Should only have one build fired.

**Comment by StrongMonkey** [comment]
Available to test in v0.5.0-alpha.3

**Comment by daxmc99** [comment]
Validated 💯 
`rio run https://github.com/rancher/rio-demo`
`rio build`
```
NAME                                          SERVICE/STACK             REVISION                                   CREATED          SUCCEED   REASON
default/default-boring-solomon4-ee709-4e40c   default/boring-solomon4   f8fab97fddc8ed5e98e45cd6373ad6feff3197f9   36 minutes ago   True
```
